recipes/pyjacqq/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "JacqQ" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sjirjies/py{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6997311506185504d71d28ccac22889bfef230bb146c4907dff5f61357c1b31b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - python >=3.4
+    - scipy
+    - numpy
+
+test:
+  imports:
+    - jacqq
+
+about:
+  home: https://github.com/sjirjies/pyJacqQ
+  license: GPL-3
+  license_family: GPL3
+  license_file: LICENSE
+  summary: |
```

Comment:
While we are waiting for @oblute for confirm she is ok being a maintainer, I would suggest to rename the current `summary` section to `description`, and use `summary: Open Soure Jacquez's Q-Statistics for Space-Time Clustering` instead.

Reply:
In general, the `summary` should be shorter, while the `description` field can be more detailed and explanatory.